### PR TITLE
Feat:Create Contribution Section

### PR DESCRIPTION
**Pull request by cp-priyanka-g**
**Video:**[New-Contribution.webm](https://user-images.githubusercontent.com/98794636/214234860-50b66d5a-cd66-4fd9-86e6-88b995f6d4cc.webm)

**Page Performance:** 25(Used gif in img tag as video looks blurr, when convert gif to video) .

**NOTE**: Change in line-height in large device as figma's line-height was looking wider.

**Tested on:**
- [x] Inspector (all iPhones and android devices like samsung, motorola)
- [x] Desktop (chrome, firefox, safari)
- [ ] Laptop / Macbook  (chrome, firefox, safari)
- [x] iPhone(safari)
- [x] Available android devices  (chrome, firefox)
- [x] iPad(safari)
- [x] Tablet (chrome, firefox)

### CODE REVIEW

File: vue-frontend/src/components/home-new/ContributionSection.vue [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+  </div>
+</template>
+<script type="module">
+import { FontAwesomeIcon } from "@fortawesome/vue-fontawesome";
+import { library } from "@fortawesome/fontawesome-svg-core";
+import { faGithub } from "@fortawesome/free-brands-svg-icons";
+import { faArrowRightLong } from "@fortawesome/free-solid-svg-icons";
+
+import complexRouting from "@/assets/images/contribution/animations/ComplexRouting.gif";
+import introShowCase from "@/assets/images/contribution/animations/introShowCase.gif";
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/home-new/ContributionSection.vue:123]: Lets keep first letter capital as others

File: vue-frontend/src/components/home-new/ContributionSection.vue [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+  data() {
+    return {
+      icon: faGithub,
+      rightArrow: faArrowRightLong,
+      isMobile: false,
+      websiteOpenSourceUrl: Config.WEBSITE_OPEN_SOURCE_URL,
+      contributions: [
+        {
+          title: "Intro Showcase view in jetpack compose",
+          description: "The missing typesafe SwiftUI navigation library.",
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/home-new/ContributionSection.vue:143]: Replace with `Highlight different features of the app using Jetpack Compose`

File: vue-frontend/src/components/home-new/ContributionSection.vue [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+          author: "Radhika S.",
+          video: introShowCase,
+          starPoints: "237",
+          link: "https://github.com/canopas/Intro-showcase-view",
+        },
+        {
+          title: "UIPilot",
+          description:
+            "An implementation of Intro Showcase from the Material Design guidelines.",
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/home-new/ContributionSection.vue:152]: Replace with `The missing typesafe SwiftUI navigation library`

File: vue-frontend/src/components/home-new/ContributionSection.vue [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+          description: "Cool animations implemented with Jetpack compose",
+          author: "Radhika S.",
+          video: JetpackComposeAnimations,
+          starPoints: "186",
+          link: "https://github.com/canopas/Jetpack-compose-animations-examples",
+        },
+        {
+          title: "Tailwind Animations",
+          description:
+            "This repository contains different animations implemented using tailwind css.",
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/home-new/ContributionSection.vue:169]: `Cool animations implemented with Tailwind CSS`

File: vue-frontend/src/components/home-new/ContributionSection.vue [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+          author: "Sumita K.",
+          video: Tailwind,
+          starPoints: "13",
+          link: "https://github.com/canopas/tailwind-animations",
+        },
+        {
+          title: "Swiftui-animations",
+          description: "Cool animations implemented with SwiftUI.",
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/home-new/ContributionSection.vue:177]: Let's remove full stop `.`

File: vue-frontend/src/components/home-new/ContributionSection.vue [hunk 6]
```diff
@@ -0,0 +1,196 @@
...
+          class="sm:tw-flex sm:tw-flex-col tw-basis-[35%] md:tw-basis-2/5 tw-my-auto tw-bg-[#FFFFFF] tw-p-[0.5rem] sm:tw-p-5 lg:tw-p-[2rem] v2-title-3-text tw-shadow-[0_0px_10px_rgba(0,0,0,0.1)]"
+        >
+          <div>
+            <div
+              @touchstart.passive="activeIndex = index"
+              :class="index % 2 != 0 ? 'sm:tw-flex-row-reverse' : ''"
+              class="tw-flex tw-flex-row tw-justify-between tw-items-center"
+            >
+              <div>
+                <span
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/home-new/ContributionSection.vue:53]: This section is wider in figma

File: vue-frontend/src/components/home-new/ContributionSection.vue [hunk 7]
```diff
@@ -0,0 +1,196 @@
...
+              <div>
+                <div>
+                  <a
+                    class="tw-flex tw-items-center tw-font-inter-bold tw-text-[0.75rem] md:tw-text-[1.25rem] tw-leading-[2.5625rem] tw-duration-300 tw-ease-in-out hover:tw-scale-[1.1] tw-hover-hover"
+                    :href="contribution.link"
+                    target="_blank"
+                    @click.native="$gtag.event('tap_library')"
+                  >
+                    <font-awesome-icon
+                      class="tw-mr-2.5 tw-w-[16px] md:tw-w-[30px] lg:tw-w-10 tw-h-[16px] md:tw-h-[30px] lg:tw-h-10 tw-text-pink-300"
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/home-new/ContributionSection.vue:71]: Width and height is 28 in desktop

File: vue-frontend/src/components/home-new/ContributionSection.vue [hunk 8]
```diff
@@ -0,0 +1,196 @@
...
+            </div>
+            <div class="tw-mt-5">
+              <span
+                class="tw-hidden md:tw-block tw-font-inter-light tw-text-[1.5rem] tw-leading-[1.875rem] tw-text-black-core/[0.6]"
+                >{{ contribution.description }}</span
+              >
+            </div>
+            <div
+              @touchstart.passive="activeIndex = index"
+              class="tw-flex tw-flex-row tw-justify-between tw-mt-[10px] md:tw-mt-[30px] tw-font-inter-regular tw-text-[1rem] tw-leading-[1.5rem] tw-text-black-core/[0.87]"
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/home-new/ContributionSection.vue:94]: Font size is 10px in mobile. Also, the design is the same as figma on mobile devices. Check design and all the sizes for the whole section

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: vue-frontend/src/components/home-new/ContributionSection.vue [hunk 9]
```diff
@@ -0,0 +1,200 @@
...
+        >
+          <img
+            :src="contribution.video"
+            :alt="contribution.title + `image`"
+            class="tw-w-full tw-h-full"
+          />
+        </div>
+
+        <div
+          class="tw-flex tw-flex-col tw-basis-[40%] md:tw-basis-[55%] lg:tw-basis-[35%] tw-my-auto tw-bg-[#FFFFFF] tw-p-2.5 sm:tw-p-4 v2-title-3-text tw-shadow-[0_0px_10px_rgba(0,0,0,0.1)]"
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/home-new/ContributionSection.vue:42]: What is the use of the `v2-title-3-text` class as we are adding font size classes separately?

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

**Review by cp-jagruti-a — APPROVED** [review]

**Review by cp-sumi-k — APPROVED** [review]

**Pull request merged**